docs/getting-started.txt

```diff
@@ -1,300 +1,311 @@
+previous example, except that a *resources* property is added with the ``resources.slots_per_trial``
+field is set to ``8`` to represent eight GPUs. The default ``slots_per_trial`` setting is ``1``. If
+``slots_per_trial`` is larger than the number of slots on an agent, set
+``resources.slots_per_trial`` to a multiple of the number of GPUs in each machine in the cluster.
+For example, if your cluster consists of 8-GPU machines, use a value that is a multiple of eight.
+For a Determined cluster deployed in the cloud, the default value for each agent is eight GPUs.
 
-.. code:: yaml
+#. Create the experiment using the distributed.yaml file:
 
-   name: mnist_pytorch
-   hyperparameters:
-     learning_rate: 0.001
-     dropout: 0.5
-     global_batch_size: 64
-     n_filters1: 40
-     n_filters2: 40
-   resources:
-     slots_per_trial: 8
-   records_per_epoch: 50_000
-   searcher:
-     name: single
-     metric: validation_error
-     max_length:
-       epochs: 64
-     smaller_is_better: true
-   entrypoint: model_def:MNistTrial
+   .. code:: bash
 
-Next, submit the experiment with the same command as above; however, our experiment configuration
-file will now be ``distributed.yaml``:
+      det experiment create distributed.yaml .
 
-.. code::
+#. Enter the cluster address in the browser.
 
-   det experiment create distributed.yaml .
+#. Accept the default determined username with no password and click **Sign In** to view the
+   dashboard.
 
-Once again, we access the experiment's trial in the browser. You will notice the loss curve is very
-similar to the single GPU experiment we previously ran. However, we reduced the time from 2 minutes
-and 16 seconds down to 1 minute and 5 seconds.
+#. Click the **Experiment** tile to view the experiment’s trial display.
 
-.. image:: /assets/images/dtrain_results@2x.jpg
-   :width: 100%
+The loss curve is similar to the single-GPU experiment in the previous exercise but the time to
+complete the trial is reduced by about half.
 
-Run Your First Hyperparameter Tuning Job
-========================================
+*********************************
+ Run a Hyperparameter Tuning Job
+*********************************
 
-Now that we have trained a model, we want to improve the model's accuracy through hyperparameter
-tuning. Typically, this requires a significant amount of time and code. Determined provides support
-for hyperparameter search through efficient execution of state-of-the-art early-stopping based
-approaches as well as seamless parallelization of these methods. To use Determined's hyperparameter
-search feature, we only have to change the experiment configuration file.
+Hyperparameter searches involve multiple trials or model variations, per experiment, to improve
+model accuracy. Typically, hyperparameter tuning takes a significant amount of time and code.
+Determined hyperparameter search efficiently executes a state-of-the-art early-stopping-based
+algorithm with seamless parallelization.
```

Comment:
I realize this is in the original, but just to note: Determined supports multiple hyperparameter search algorithms, not all of which use early stopping.

Reply:
Deleted; unhelpful marketing hype.